src/main/java/gregtech/loaders/recipe/CraftingComponent.java

```diff
@@ -250,6 +250,7 @@ public static void initializeComponents() {
                 {6, new UnificationEntry(OrePrefix.plate, Materials.RhodiumPlatedPalladium)},
                 {7, new UnificationEntry(OrePrefix.plate, Materials.NaquadahAlloy)},
                 {8, new UnificationEntry(OrePrefix.plate, Materials.Livermorium)},
+                {9, new UnificationEntry(OrePrefix.plate, Materials.Neutronium)},
```

Comment:
Instead of adding neutronium here, we should stop registering the MAX tier versions of diodes instead.

Reply:
This is not MAX, this is UHV